README.md

```diff
@@ -1,2 +1,33 @@
 # adp-flux-core
-This will be the main repository used for configuring GitOps. It will contain the manifests to deploy the core services to the AKS Cluster e.g. for secrets management, NGINX Ingress Controller.
+Flux v2 configuration for ADP cluster core components amd services
+
+## Introduction
+This repository contains the Flux configuration for the ADP cluster core components and services. The repository is structured to support multiple environments (snd, dev, tst, pre, prd) and multiple clusters per environment. The repository is structured as follows:
+
+## Repository Structure
+
+* `clusters` - Flux configuration for the Flux System (controllers etc) per environment
+* `infra` - Flux configuration for the core services e.g. Nginx Plus per environment
+* `core` - Contains the manifests for the core service (HelmRelease, HelmRepository)
+```
+
+├── clusters
+│   ├── snd      
```

Comment:
is it worthwhile doing snd1, tst1, pre1, prd1, dev1, snd2, snd3 etc?

in docs & folder names - to match Azure env names exactly @kbabigumira  @ppotturi

Reply:
clusters/snd1/01 and clusters/snd2/01 ?
 
 I.e. within a subscription, we could have 2 AKS Clusters (01 and 02), but that would still be SND1 or PRD1, for example.

Reply:
Are we going to have PRD1 and PRD2 environments? I think the PRD environment can have 10 and 02 clusters.

Reply:
reason for snd folder is, all SND (1,2& 3) environments can always be the same config.

Reply:
Design decision. Do we need different SND or TST env configurations? Wouldn't cluster 01 in SND1 have an update that SND2 Cluster 01 wouldn't ? for a bit?

Reply:
Environments can have the same config, but could they be different? are we differentiating SND1 2 and 3 via cluster number only?

Reply:
As discussed, we are keeping the folder structure as it is.